src/ds_caselaw_utils/data/court_names.yaml

```diff
@@ -323,21 +305,42 @@
       link: https://www.gov.uk/courts-tribunals/chancery-division-of-the-high-court
       selectable: false
       listable: false
+- name: lower_courts
+  display_name: "Lower Courts"
```

Comment:
Should we group these "lower courts" since they are not technically grouped?

For reference: https://www.judiciary.uk/structure-of-courts-and-tribunals-system/

Reply:
@jacksonj04 thoughts?

Reply:
I would say these should in an ideal world be top-level, because as you say they're not really a grouping and we should lean towards reflecting the objective truth where we can.

Reply:
So having looked into how this yaml file is loaded and used, it seems like we do depend on the top level being some sort of group. So for now, this makes sense, but yeah like you said we could decide to reflect the truth a bit more and not allow the data structure to not need to have a top level grouping and instead allow both court groups and individual courts at the same level of the tree.

![Screenshot 2023-06-27 at 09 24 07](https://github.com/nationalarchives/ds-caselaw-utils/assets/42998618/eb18dfee-aebd-4b6c-8ff8-75796aaea32e)

Reply:
In the PU, we of course loop through lists of all court to display the courts to filter by, but the only place we currently access groups of courts is in the about this service view. Here we use the display name, which in this case will be "lower courts", so we should think about whether we will want to present that like that there and if not, whether the extra work to modify it is better done here in one place rather than potentially in 2 or more places (about this service and the court filtering pages)

![Screenshot 2023-06-27 at 09 36 52](https://github.com/nationalarchives/ds-caselaw-utils/assets/42998618/4beb9257-dbb9-4084-b4f3-393260b65ad0)